client/components/web-preview/content.jsx

```diff
@@ -383,7 +383,7 @@ export default class WebPreviewContent extends Component {
 								src="about:blank"
 								onLoad={ () => this.setLoaded( 'iframe-onload' ) }
 								title={ this.props.iframeTitle || translate( 'Preview' ) }
-								fetchpriority={ fetchPriority ? fetchPriority : undefined }
+								fetchPriority={ fetchPriority ? fetchPriority : undefined }
```

Comment:
it's lowercase in the DOM but as a JSX attribute, it would be camel-case like the others. It's still an experimental attribute which is why it's not in the React repo. React does output it as lowercase though. 

![image](https://user-images.githubusercontent.com/6851384/178286703-b5d4b524-780a-443c-9ba2-53e9db020ddd.png)

So the argument for making this change is that if `fetchpriority` becomes standard our code will be correct and the case is consistent with other JSX attributes. We also don't break anything as a result of making the change now.

Reply:
Thanks for confirming! I wasn't able to work out how this specific case gets handled in the UI -- I wanted to make sure the attribute won't get transformed to `fetch-priority` due to the camel case usage.

Reply:
@mpkelly It looks like this now throws a warning for `fetchPriority`. Here's what I'm seeing currently on trunk:

- Using the development version of Calypso (build it locally using `yarn start`) so you can see the React warnings
- Visit `/view/{site}`
- I now see the following warning in the JS console

```
Warning: React does not recognize the `fetchPriority` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `fetchpriority` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

Reply:
Hi @creativecoder , yes, this attribute is not officially supported, so React still outputs an error regardless. What I did was change the case to match the JSX convention. If this attribute ever does become official then this warning will go away. I verified that React converts from camel to lower even for unknown attributes.

Reply:
If I revert `fetchPriority=` to `fetchpriority=`, the console warning goes away.

So what's the advantage of using `fetchPriority=`? If it's being converted to `fetchpriority` anyway, isn't it better to not have the distraction of the console warning?

Reply:
>So what's the advantage of using fetchPriority=? If it's being converted to fetchpriority anyway, isn't it better to not have the distraction of the console warning?

Yeah, the goal was to clear console warnings, so I don't think this should have been included 😃 .There are [three locations where we use camel-case](https://github.com/Automattic/wp-calypso/search?q=fetchPriority), including this one, so maybe a small PR could be made to clear those. If you think it's worth it then I will do that tomorrow.

Reply:
It's easy enough to address, so it does seem worth fixing! https://github.com/Automattic/wp-calypso/pull/66070